cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
@cpennington I can't recall it either but I asked you question about it on my original django-sudo PR here https://github.com/edx/edx-platform/pull/7404#issuecomment-95612166 and for some reason I never converted ```base.html``` into django template.

I don't remember exactly why we never converted it.